mlflow/environment_variables.py

```diff
@@ -92,10 +92,33 @@ def get(self):
 #: (default: ``None``). If None, ``google.cloud.storage.constants._DEFAULT_TIMEOUT`` is used.
 MLFLOW_GCS_DEFAULT_TIMEOUT = _EnvironmentVariable("MLFLOW_GCS_DEFAULT_TIMEOUT", int, None)
 
-#: Specify whether to disable model logging and loading via mlflowdbfs.
-#: (default: ``None``)
+#: Specifies whether to disable model logging and loading via mlflowdbfs.
+#: (default: `None`)
```

Comment:
```suggestion
#: (default: ``None``)
```

Reply:
@harupy 
What is the difference between single and double quotes here?

Reply:
<img width="843" alt="image" src="https://user-images.githubusercontent.com/17039389/185079630-7c40fedc-08d1-4160-96ad-880904161f5e.png">

Reply:
thanks!